lib/pretty/pretty.mli

```diff
@@ -1,5 +1,8 @@
 (** A library for declarative UI definition. *)
 
+(** Common styles for GitHub TUI. *)
+module Style = Style
```

Comment:
I'm no longer exporting `Style` from both `Pretty.Doc` and `Pretty.Layout`. It was an unfortunate coincidence anyway.

Now `Style` should be used as `Pretty.Style`